packages/components/src/components/hds/breadcrumb/item.ts

```diff
@@ -33,21 +29,25 @@ export interface HdsBreadcrumbItemSignature {
 }
 
 export default class HdsBreadcrumbItem extends Component<HdsBreadcrumbItemSignature> {
-  @tracked linkToExternal: LinkTo | null = null;
+  static linkToExternal: LinkTo | null = null;
 
-  constructor(owner: Owner, args: HdsBreadcrumbItemSignature['Args']) {
-    super(owner, args);
+  /**
+   *
+   * @param linkToExternal
+   * @type LinkTo | null
+   * @default null
+   */
+  get linkToExternal(): LinkTo | null {
+    const component = HdsBreadcrumbItem.linkToExternal;
+    if (component === null) {
+      assert(
+        `HdsBreadcrumbItem: You attempted to use an external link without configuring HDS with an external component. Please add this in your app.js file:
 
-    // we want to avoid resolving the component if it's not needed
-    if (args.isRouteExternal) {
-      void this.resolveLinkToExternal();
+import { HdsBreadcrumbItem } from @hashicorp/design-system-components/components;'
```

Comment:
@shleewhite do you think we can simply replace the link inside the `HdsBreadcrumbItem` with the `HdsInteractive` components, so all this logic is centralized in a single place/component?